app/assets/stylesheets/views/_landing_page/card.scss

```diff
@@ -0,0 +1,44 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-b-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: govuk-colour("dark-blue");
+  color: govuk-colour("white");
+}
+
+.app-b-card__textbox {
+  padding: govuk-spacing(6) govuk-spacing(7);;
+  background-color: govuk-colour("dark-blue");
+  color: govuk-colour("white");
+
+  .gem-c-govspeak h2 {
+    @include govuk-font($size: 27, $weight: bold, $line-height: 34px );
+  }
+
+}
+
+.app-b-card__figure {
+  border: 1px solid govuk-colour("mid-grey");
+  margin: auto 0 0;
+  background-color: govuk-colour("white");
+}
+
+.app-b-card__image {
+  display: block;
+  width: 100%;
+  max-width: 100%;
```

Comment:
Does the image need both `max-width` and `width` 100%? Seems like `width: 100%` would be enough?

Reply:
Good point, I've removed the max width.